projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/recommendations/Pagination.svelte

```diff
@@ -0,0 +1,131 @@
+<script lang="ts">
+	import { imageStore } from '../ApiMock';
+
+	// "-1" is replaced by "..." when rendering the pagination
+	const MORE_ICON = -1;
+
+	function slidingWindow( currentPage, maxPage ) {
```

Comment:
This function needs a comment describing its purpose, and preferably a return type specified in TypeScript to make it clear.